frontend/src/features/properties/selector/MapSelectorModalContainer.tsx

```diff
@@ -1,77 +0,0 @@
-import { Button } from 'components/common/form';
```

Comment:
I don't see a replacement for this, is there still a way to pop up a window with the selector embedded into it? this may be required for future stories.

Reply:
There is not one at the moment. The refactor render this one unusable, and since is not being used atm I did not refactor this one.

Reply:
same comment then, lets meet offline so I can understand why the refactoring was necessary.